code/modules/clothing/suits/ego_gear/he.dm

```diff
@@ -166,33 +174,33 @@
 								FORTITUDE_ATTRIBUTE = 40
 								)
 
-/obj/item/clothing/suit/armor/ego_gear/brick
+/obj/item/clothing/suit/armor/ego_gear/he/brick
 	name = "yellow brick"
 	desc = "The heavy dress weighs you down, smacking against your knees."
 	icon_state = "brick"
-	armor = list(RED_DAMAGE = -20, WHITE_DAMAGE = 50, BLACK_DAMAGE = 30, PALE_DAMAGE = 10)
+	armor = list(RED_DAMAGE = 30, WHITE_DAMAGE = 50, BLACK_DAMAGE = -20, PALE_DAMAGE = 10)
```

Comment:
ya should probably mention this change, if its intentional

Reply:
Uh oh, random EGO rebalance??

Reply:
Unused armor, same numbers as another EGO